src/game/common/rts/buildinfo.cpp

```diff
@@ -25,18 +25,18 @@ BuildListInfo::BuildListInfo() :
     m_script(),
     m_health(100),
     m_whiner(true),
-    m_repairable(false),
-    m_sellable(true),
+    m_sellable(false),
+    m_repairable(true),
```

Comment:
Curious, why have we flipped the boolean values here for m_sellable and m_repairable?

Reply:
@jonwil

Reply:
Jonwil says this is correct